superset-frontend/src/components/TableView/TableView.tsx

```diff
@@ -51,7 +52,16 @@ const EmptyWrapper = styled.div`
 
 const TableViewStyles = styled.div<{
   isPaginationSticky?: boolean;
+  scrollTable?: boolean;
 }>`
+  ${({ scrollTable, theme }) =>
+    scrollTable &&
+    `
+    height: 300px;
```

Comment:
That's a good idea. However, there is some calculation to do to make sure the height is just right for the modal to avoid showing its own scrollbar. I'll keep this in mind for an enhancement and provide a separate PR for the sake of getting this improvement in asap